src/OrbitGl/AccessibleInterfaceProvider.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_ACCESSIBLE_INTERFACE_PROVIDER_H_
+#define ORBIT_GL_ACCESSIBLE_INTERFACE_PROVIDER_H_
+
+#include "OrbitAccessibility/AccessibleInterface.h"
+
+namespace orbit_gl {
+
+/* Base class to provide accessibility in the capture window. */
+class AccessibleInterfaceProvider {
+ public:
+  explicit AccessibleInterfaceProvider(AccessibleInterfaceProvider* parent) : parent_(parent) {}
+
+  [[nodiscard]] orbit_accessibility::AccessibleInterface* GetOrCreateAccessibleInterface();
+  [[nodiscard]] const orbit_accessibility::AccessibleInterface* GetAccessibleInterface() const {
+    return accessibility_.get();
+  }
+
+  [[nodiscard]] virtual AccessibleInterfaceProvider* GetParent() const { return parent_; }
```

Comment:
You don't really need this here anymore, do you? It's overridden in `CaptureViewElement` with a different return type, but since `TimeGraph` has its own accessibility class and special handling of this case, there's no need to declare it here.

Reply:
GlCanvas is using it. I prefer to have it here than in GlCanvas.

Reply:
You are right, thanks. I was sure that GlCanvas needed the GetParent() method. My bad.